### ISSUE TITLE
sendKeys(protractor.Key.TAB) has no effect

### ISSUE THREAD

**Issue by tempoc** [angular/protractor#2370]
A test that tabs through form fields and routinely works suddenly ceased to do so in the past couple of days. The relevant source:

```javascript
beforeEach(function () {
    firstLabel.click();
    firstField.click();
});

it("should become invisible when tabbed out", function () {
    firstField.sendKeys(protractor.Key.TAB);

    expect(firstField.isDisplayed()).toBeFalsy();
});
```

`protractor.Key.TAB` is defined and in the console prints out the equivalent of `'\uE004'`

The test is running on the latest version of Chrome.
Some of the npm devDependencies are:
```
gulp-protractor@0.0.11 node_modules/gulp-protractor
├── dargs@0.1.0
├── async@0.7.0
├── event-stream@3.1.7 (stream-combiner@0.0.4, duplexer@0.1.1, from@0.1.3, pause-stream@0.0.11, map-stream@0.1.0, split@0.2.10, through@2.3.8)
├── gulp-util@2.2.20 (lodash._reinterpolate@2.4.1, minimist@0.2.0, chalk@0.5.1, vinyl@0.2.3, lodash.template@2.4.1, through2@0.5.1, dateformat@1.0.11, multipipe@0.1.2)
└── protractor@2.1.0 (jasminewd@1.1.0, jasminewd2@0.0.5, html-entities@1.1.3, saucelabs@0.1.1, q@1.0.0, minijasminenode@1.1.1, optimist@0.6.1, adm-zip@0.4.4, glob@3.2.11, jasmine@2.3.1, source-map-support@0.2.10, accessibility-developer-tools@2.6.0, request@2.36.0, lodash@2.4.2, selenium-webdriver@2.45.1)
```

And output from jar updates reads:
```
Updating selenium standalone
downloading https://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar...
Updating chromedriver
downloading https://chromedriver.storage.googleapis.com/2.15/chromedriver_mac32.zip...
chromedriver_2.15.zip downloaded to [path]/node_modules/gulp-protractor/node_modules/protractor/selenium/chromedriver_2.15.zip
selenium-server-standalone-2.45.0.jar downloaded to [path]/node_modules/gulp-protractor/node_modules/protractor/selenium/selenium-server-standalone-2.45.0.jar
```

**Comment by tempoc** [comment]
More info: This works in Firefox but not in Chrome
```javascript
describe('Test', function () {
  it('should browse to google', function () {
    browser.ignoreSynchronization = true;
    browser.driver.get('https://www.google.de');
    expect(browser.getCurrentUrl()).toEqual('https://www.google.de/');
  });
  it('should unfocus the search field', function () {
    var search = element(by.name('q'));
    search.sendKeys(protractor.Key.TAB);
    browser.sleep(3000); // 3s to take a look ;)
  });
});
```

**Comment by unkillbob** [comment]
Can confirm we're also having this issue. It doesn't seem to work using action sequence either:

```js
browser.actions().sendKeys(protractor.Key.TAB).perform()
```

**Comment by jlin412** [comment]
I see the problem in FF but not in Chrome.

**Comment by jschlot** [comment]
just out of curiosity - can you try to detect a blur() instead of tabbing out?

**Comment by ashaffer** [comment]
+1 also having this issue, and notably have not updated protractor either.  It seems this is either a chrome update or a webdriver update of some kind.

**Comment by ghiscoding** [comment]
I got the same problem and this was working couple days ago, I believe as well that it's the Chrome update that is now causing this issue. 

My Chrome version is Version 44.0.2403.89 m (64-bit)

**Comment by CharlieTruong** [comment]
I've also just experienced this issue.

**Comment by hirtenfelder** [comment]
I have the same problem with Angular Protractor 2.1.0. With update to chrome version 44.0.2403.89 m WebElement.sendKeys(protractor.Key.UP) will not work anymore.

Seems to be an issue not only with protracotr:

https://code.google.com/p/chromedriver/issues/detail?can=2&start=0&num=100&q=&colspec=ID%20Status%20Pri%20Owner%20Summary&groupby=&sort=&id=1159

**Comment by andrew-luhring** [comment]
with elementExplorer:
```js
browser.ignoreSynchronization = true
browser.get('https://github.com/angular/protractor/issues/new')
//this works:
element(by.name('q')).sendKeys('derp');
//this works:
element(by.name('q')).sendKeys(protractor.Key.BACK_SPACE)
//this doesn't:
element(by.name('q')).sendKeys(protractor.Key.TAB)
```
Is this not how you're supposed to get protractor to tab away from elements?
It used to work:
http://stackoverflow.com/questions/28410273/how-to-send-keyboard-keys-in-protractor-like-tab
but my tests started failing this week after updating webdriver.

( Celebrate the little victories - good job me for not creating a duplicate issue  🙌 )

**Comment by virtser** [comment]
Oh, we experience the issue as well, it breaks our tests! :(

**Comment by CThuleHansen** [comment]
Breaking tests here too :/

**Comment by aaronpmishkin** [comment]
Breaking my tests as well.

**Comment by mfsiabhishek** [comment]
Same here, tests are broken on Chrome 44.

**Comment by ashaffer** [comment]
This is now fixed in the latest version of Chrome!  Finally.

**Comment by CThuleHansen** [comment]
Thank you for heads up @ashaffer

**Comment by tempoc** [comment]
Fixed with https://code.google.com/p/chromium/issues/detail?id=513768

**Comment by Satbeen** [comment]
Again broken in chrome version 53 with protractor version 4.0.0 [also with 4.0.9]. Was working wit 52nd version of chrome. Does other also face the same problem?

**Comment by calystoramesh** [comment]
yes, I got the same problem

**Comment by MariuszKutek** [comment]
Yes, I have the same problem.

**Comment by jfranki** [comment]
Same problem with Protractor 4.0.9 and Chrome 53.0.2785.116 m (64-bit).

**Comment by Narretz** [comment]
I am on Chrome Version 53.0.2785.143 m and see the same problem. Has somebody opened an issue for this on Chrome yet?

**Comment by Narretz** [comment]
Issue https://bugs.chromium.org/p/chromium/issues/detail?id=643617 seems to refer to the problem, and the last comment says that this is fixed in Chromedriver 2.2.24, which has been added in Webdriver Manager (but not released), so Protractor currently uses an older version of Chrome Driver.
So in order to get this fixed (I think), 
- Webdriver Manager must cut a release
- Protractor must update the WM dependency
- Protractor must cut a release

pinging @cnishina
